test/unit/downloader.test.ts

```diff
@@ -96,6 +100,64 @@ test('Downloader class', async (t) => {
 
     rimraf.sync(cacheDir);
 
+    const isPngFile = await downloader.isImageUrl('https://bm.wikipedia.org/static/images/project-logos/bmwiki-2x.svg.png');
+    t.assert(isPngFile, 'Checked Image type: png');
+
+    const isJpgFile = await downloader.isImageUrl('https://bm.wikipedia.org/static/images/project-logos/bmwiki-2x.JPG');
+    t.assert(isJpgFile, 'Checked Image type: jpg');
+
+    const isSvgFile = await downloader.isImageUrl('https://bm.wikipedia.org/static/images/project-logos/bmwiki-2x.svg');
+    t.assert(isSvgFile, 'Checked Image type: svg');
+
+    const isJpegFile = await downloader.isImageUrl('https://bm.wikipedia.org/static/images/project-logos/bmwiki-2x.JPEG');
+    t.assert(isJpegFile, 'Checked Image type: jpeg');
+
+    const isgifFile = await downloader.isImageUrl('https://bm.wikipedia.org/static/images/project-logos/bmwiki-2x.gif');
+    t.assert(isgifFile, 'Checked Image type: gif');
+
+    const isnotImage = await downloader.isImageUrl('https://en.wikipedia.org/w/api.php?action=query&meta=siteinfo&format=json');
```

Comment:
This are only the simple cases, unit test should test edge case with:
* undef value
* empty string value
* no extension
* maybe extension pattern at the end like "asdfasdf.webm.png" and "asdfasdf.png.svg".
* uppercased extension
*